src/Microsoft.AspNetCore.Hosting/Internal/HostingApplication.cs

```diff
@@ -64,21 +74,65 @@ public Context CreateContext(IFeatureCollection contextFeatures)
                 // Non-inline
                 LogRequestStarting(httpContext);
             }
-
             if (diagnoticsEnabled)
```

Comment:
It could be a valid scenario, but probably useless.
Why would you want to log only start of the request without end of it (i.e. result and duration)?